.changeset/afraid-coats-call.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-dropdown": minor
```

Comment:
**suggestion:** This is more of a `patch` version as you are fixing a bug in the existing implementation.

More of that here: https://khanacademy.atlassian.net/wiki/spaces/WB/pages/2217640195/Versioning+and+releasing+Wonder+Blocks+packages#Patch-changes-0.0.1